babel.config.js

```diff
@@ -25,7 +25,9 @@ module.exports = (api) => {
   ];
 
   const buildPlugins = clean([
-    '@babel/plugin-proposal-class-properties',
+    ['@babel/plugin-proposal-class-properties', { loose: true }],
+    ['@babel/plugin-proposal-private-methods', { loose: true }],
+    ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
```

Comment:
I'd rather do this for storybook alone, so looking into if it can have its own babel config